### ISSUE TITLE
Outdated example

### ISSUE THREAD

**Issue by KokaKiwi** [hyperium/hyper#895]
Well, simply put, example seems to be really outdated with actual `hyper` documentation, which is really disturbing for newcomers...

**Comment by Stargateur** [comment]
I try it, i don't know if i do it well
``` rust
#![deny(warnings)]
extern crate hyper;
extern crate env_logger;
extern crate num_cpus;

use hyper::server::{Server, Handler, Request, Response};
use hyper::net::{HttpListener};

static PHRASE: &'static [u8] = b"Hello World!";

struct Hello;

impl Handler for Hello {
    fn handle(&self, _: Request, res: Response) {
        res.send(PHRASE).unwrap();
    }
}
fn main() {
    env_logger::init().unwrap();
 
    let listener = HttpListener::new("127.0.0.1:3000").unwrap();
    let mut handles = Vec::new();

    for _ in 0..num_cpus::get() {
        let listener = listener.clone();
        handles.push(::std::thread::spawn(move || {
            Server::new(listener)
                .handle(Hello).unwrap();
        }));
    }
    println!("Listening on http://127.0.0.1:3000");

    for handle in handles {
        handle.join().unwrap();
    }
}
```

**Comment by seanmonstar** [comment]
I'm sorry, I'm not quite sure what you mean. Which example are you referring to?

**Comment by KokaKiwi** [comment]
The ones in the `examples` folder

**Comment by KokaKiwi** [comment]
Aah my bad.

The confusion came from the fact that the examples on the `master` branch doesn't work for the *published* version (aka the branch `0.9.x`) (which is comprehensible since there's a rework in progress)

This issue should be closed as `not-a-bug` :stuck_out_tongue:

**Comment by seanmonstar** [comment]
Sorry about the confusion!
